src/view/App.svelte

```diff
@@ -1,20 +1,26 @@
 <script>
-	import {onMount} from 'svelte'
-	import {initConnection} from '../services/connection'
-	import {connection} from '../data/store'
-	import Login from "./components/Login/Login.svelte"
-	import Tickets from './components/Tickets/Tickets.svelte'
-	import './styles.css'
+  import {onMount} from "svelte";
+  import {initConnection} from "../services/connection";
+  import {connection, originAccount} from "../data/store";
+  import Login from "./components/Login/Login.svelte";
+  import Tickets from "./components/Tickets/Tickets.svelte";
+  import "./styles.css";
 
-	onMount(async () => {
-		connection.update($connection => {
-			$connection = initConnection(process.env.CLUSTER_ENDPOINT)
-			return $connection
-		})
-	})
+  window.addEventListener("message", event => {
```

Comment:
why not add this directly to the Login component? I don't think we would need state in that case at all